lib/SMS.rb

```diff
@@ -0,0 +1,27 @@
+require 'twilio-ruby'
+
+class SMS
```

Comment:
I think sending the message and defining the contents of the message are two different responsibilities. You've broken them into different methods (good), but I think that the second one doesn't really even belong in this class.